### ISSUE TITLE
N

### ISSUE THREAD

**Issue by wellsana** [kaviraj07/Hospital-Management-System-RMI#1]
Hey man ! Iam trying toopen the hospitalImplemenetations projeect on netbeans but it doesnt work . should i use another IDE?

**Comment by kaviraj07** [comment]
This project was built using Eclipse IDE. So use that IDE to open it or try importing the project as an 'eclipse project' when using NetBeans.

**Comment by wellsana** [comment]
thx for responding!  but i keep getting errors . are there some libraries that i need to add to the project? 
like in theses lines : import static hospitalImplementations.environment.port;
import static hospitalImplementations.environment.server_address;

**Comment by kaviraj07** [comment]
As far as I remember you would need the JDBC driver for database connection. Check the code for any other libraries that you don't have. 

For the line you mentioned: " import static hospitalImplementations.environment.port; " these are just imports and the files are already there. You might need to configure your ports in the environment.java file

**Comment by wellsana** [comment]
Sorry to disturb you but it seems like the files dont exist in the project  and in the client side the file hospitalInterfaces doesnt exist also .

**Comment by kaviraj07** [comment]
Yes, indeed, after examination, the hospitalInterfaces seems to be missing. As this is an old project, I don't think I have the files for this. The implementation is there and you just have to create the interface files with the method calls.
